### ISSUE TITLE
run lint before tmt run

### ISSUE THREAD

**Issue by thrix** [psss/tmt#73]
Seems it would be nice if we would first check if the metadata are correct before trying to run them, so we can nicely report to user the errors. We can continue in case of warnings, but for errors we should not try to run ...

**Comment by psss** [comment]
Yeah, that could prevent some unexpected behaviour. Adding to the backlog with the `should` priority.
